src/components/custom/signup.tsx

```diff
@@ -92,15 +92,30 @@ const Signup = (props: SignupProps) => {
         return error_message
     }
 
+    // this method is used to prevent DOM XSS invulnerability by removing HTML DOM elements in the email input
+    function escapeHtml(str: string) {
+        return str.replaceAll(/[&<>"'\/]/g, c => {
+            switch (c) {
+                case '&':
+                case '<':
+                case '>':
+                case '"':
+                case "'":
+                    return ''
+            }
+        });
+    }
```

Comment:
This function now disallow users to enter the following symbols: `& < > " '` to prevent XSS DOM injection